src/gc-stacks.c

```diff
@@ -81,7 +81,7 @@ static void *malloc_stack(size_t bufsz) JL_NOTSAFEPOINT
     return stk;
 }
 
-static void free_stack(void *stkbuf, size_t bufsz) JL_NOTSAFEPOINT
+void free_stack(void *stkbuf, size_t bufsz) JL_NOTSAFEPOINT
```

Comment:
Why no longer `static`? If MMTk is also using this function (which looks like the most reasonable guess), then we should consider adding a declaration to a header such as `julia_internal.h` (so you don't need to re-declare in MMTk itself).